xinference/tests/test_client.py

```diff
@@ -189,222 +181,6 @@ def test_client_custom_model(setup):
     assert custom_model_reg is None
 
 
-@pytest.mark.skipif(os.name == "nt", reason="Skip windows")
```

Comment:
I see you've moved the tests to xinference-client. But these tests are not just unittests for the client, they also serve as end to end test for the system. Although some test cases are also covered by `xinference/core/tests/test_restful_api.py`, parallel generation and embedding creation are not.

To ensure the test quality, I think we can keep them for now. And once `test_restful_api.py` covers all the end to end test cases, they could be savely removed.